pkg/ytconfig/master.go

```diff
@@ -37,6 +41,7 @@ type MasterServer struct {
 	CypressManager   CypressManager   `yson:"cypress_manager"`
 	PrimaryMaster    MasterCell       `yson:"primary_master"`
 	SecondaryMasters []MasterCell     `yson:"secondary_masters"`
+	DiscoveryServers DiscoveryServers `yson:"discovery_server"`
```

Comment:
Let's keep the naming consistent with the yson config name. So DiscoveryServer without the plural.